test/parallel/test-http2-ping.js

```diff
@@ -71,12 +71,60 @@ server.listen(0, common.mustCall(() => {
         assert.deepStrictEqual(payload, ret);
       })));
     }
+
     // Only max 2 pings at a time based on the maxOutstandingPings option
     assert(!client.ping(common.expectsError({
       code: 'ERR_HTTP2_PING_CANCEL',
       type: Error,
       message: 'HTTP2 ping cancelled'
     })));
+
+    // should throw if payload is not of type ArrayBufferView
```

Comment:
Nit: please always use a capital letter as first letter in a comment and punctuate the comments as well.